packages/db-objects-redis/src/lib/objects/objectsInRedisClient.ts

```diff
@@ -579,25 +656,32 @@ class ObjectsInRedisClient {
                 }
 
                 this.sub.on('ready', async () => {
+                    if (!this.sub) {
+                        // client gone while ready emitted, can maybe not happen but ts is happy
```

Comment:
Yes you are right. The question remains are we sure it cannot happen in between on an emit? I would keep the guard.

Reply:
If you alias it as a local constant, the constant will always be defined, no matter if `this.sub` gets nulled in the meantime. Wonder if that would cause other problems though.

Reply:
Yeah, but maybe it would then throw something like connection closed, which would again need to be caught